### ISSUE TITLE
Size of the docker container

### ISSUE THREAD

**Issue by sjunges** [moves-rwth/storm#567]
In our dockerfile, we currently use multiple layers with multiple compile commands. This generally leads to big docker containers. An open question is whether we can make these docker containers smaller.

**Comment by volkm** [comment]
There is definitely some room for improvement by combing commands in one layer.
Another possible option could be to simply install Storm via CMake and then remove all the build artifacts.

**Comment by sjunges** [comment]
I like this second option as being particularly clean...

**Comment by volkm** [comment]
I agree that installing within the Docker container is the best option.

This issue then depends on #527.
